test/utilization.jl

```diff
@@ -0,0 +1,46 @@
+
+### UTIL Restoration Tests
+@testset "UTIL" begin
```

Comment:
I would also suggest adding a test that runs restoration redispatch on a util ordering, to make sure that workflow is also operational.

Reply:
done!